woocommerce/woocommerce
# Update CYS color palettes and fonts based on the new Dotcom release

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #40986.

This PR updates the color palettes and fonts based on the new Dotcom release. 

Besides, this PR also fixes bugs:

1. Remove `externalFontFamilies.length` check. That unnecessary check causes an error where the font is not loaded when selecting a single font (such as Inter + Inter).
2. Add the "Creatio" basic style to color palette options to fix the weird button/text colors because TT3 doesn't have these styles.

Before:
![Screenshot 2023-10-25 at 16 12 27](https://github.com/woocommerce/woocommerce/assets/4344253/df70adb6-7302-4d73-89e8-8459a5cd612f)

After:
![Screenshot 2023-10-25 at 16 12 46](https://github.com/woocommerce/woocommerce/assets/4344253/25b2b1e5-bcd7-4176-ad41-42ffae12f312)


#### Fonts

1. Contemporary:

- Inter, Inter (default)
- Albert sans, Lora
- Plus Jakarta Sans, Plus Jakarta Sans
- Montserrat, Arvo
- Commissioner, Crimson Pro
- Rubik, Rubik

4. Classic:

- Bodoni Moda, Overpass (default)
- Fraunces, Libre Franklin
- Newsreader, Newsreader
- Raleway, Cormorant
- Playfair Display, Fira Sans
- Libre Franklin, EB Garamond


5. Bold:

- Plus Jakarta Sans, Plus Jakarta Sans (default)
- Rubik, Inter
- Albert sans, Lora
- Montserrat, Arvo
- Raleway, Cormorant
- Inter, Inter

#### Colors

We have some color palettes that are not accessible. Veronica has highlighted them with the **red dot**. We don't include them in our CYS assembler.

Figma: Mrk6SERPZ4KrFHSjM0a8TK-fi-6948_115242



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

This feature is part of the larger customize your store development and behind a feature flag, hence QA review can be deferred until the call for testing before release.

1. Create a new WooCommerce installation with this version.
2. Make sure to enable `customize-store` feature flag
4. Install and activate [WooCommerce Blocks nightly build](https://github.com/woocommerce/woocommerce-blocks/releases/tag/nightly) 
6. Connect your site with Jetpack
7. Go to `WooCommerce -> Home` and click `Customize Store` task
8. Click `Design with AI` button
9. Continue to Look and Feel step
10. Choose`Contemporary`
11. Follow through the flow all the way till the assembler hub shows up
12. Click on `Change fonts`
13. Observe that `Inter + Inter` is default font.
14. Observe that the 6 'Contemporary' fonts are shown,
15. Confirm that the font displays correctly after selecting each font
16. Go to "change your colors"
17. Confirm 9 'Contemporary' colors are shown 
18. Select color palettes and make sure the button colors look correct
19. Repeat steps 6-11 with the other look and feel choices



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
